### DISCUSSION TITLE
Description of OpenTherm topics

### DISCUSSION THREAD

**Discussion by pticavorobey** [arendst/Tasmota#16870] [Q&A] [answered]
Hello friends.
I compiled the tasmota firmware from OpenTherm. I connected it to my gas boiler and I get a set of parameters in the MQTT. However, I am having difficulty comparing these parameters. Please help.

`{
   "Time":"2022-10-20T15:25:53",
   "OPENTHERM":{
      "conn":"BUSY",
      "settings":3,
      "SLAVE":{
         "FAULT":0,
         "CH":1,
         "DHW":0,
         "FL":1,
         "COOL":0,
         "CH2":0,
         "DIAG":0,
         "RAW":3221225482
      },
      "BTMP":{
         "FAULT":0,
         "REQ":45.0,
         "ACT":45.0
      },
      "HWTMP":{
         "REQ":50.0,
         "ACT":50.0
      },
      "ASFF":{
         "FC":0,
         "OFC":0
      },
      "FLM":0.0,
      "TB":61.0, 
      "TDHW":59.0,
      "DHWS":50.0, 
      "TMAX":0.0,
      "OT10":{
         "lo":0,
         "hi":0
      },
      "OT19":0.0
   }
}`

I couldn't recognize all the options.

**Comment by sfromis** [comment, marked as answer]
Do not cross-post, already linked a pointer on Discord
https://github.com/arendst/Tasmota/blob/development/tasmota/tasmota_xsns_sensor/xsns_69_opentherm_protocol.ino#L58-L255
